### PR TITLE
add warning message to vcs troubleshooting page about reauthorizing

### PR DESCRIPTION
**Pull request by mjyocca**
### What

Changing VCS troubleshooting page to include warning about `Reauthorize` incompatibility with OAuth Clients managed by the TFE Provider.

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/vcs/troubleshooting.mdx [hunk 1]
```diff
@@ -106,6 +106,8 @@ Terraform Cloud does not allow you to change the API URL for an existing VCS con
 
 If a VCS OAuth connection breaks, you can reauthorize an existing VCS provider while retaining any VCS connected resources, like workspaces. We recommend only using this feature to fix broken VCS connections. We also recommend reauthorizing using the same VCS account to avoid permission changes to your repositories. 
 
+~> **Important:** Reauthorizing is incompatible for OAuth Clients managed by the [TFE Provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/oauth_client).
```
Comment (laurapacilio) [inline website/docs/cloud-docs/vcs/troubleshooting.mdx:109]: Question - in this case, is there anything else they can do, or are they out of luck? If there is an alternative, we should tell them/ at least give them a path forward when something is broken.

```suggestion
~> **Important:** Reauthorizing is not available when the [TFE Provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/oauth_client) is managing the OAuth Client.
```

Comment (mjyocca) [reply website/docs/cloud-docs/vcs/troubleshooting.mdx:109]: @laurapacilio Great point!

The TFE-provider/terraform equivalent would be to rotate the `oauth_token` argument for the `tfe_oauth_client` resource.

I'll go ahead and add that in.

Comment (laurapacilio) [reply website/docs/cloud-docs/vcs/troubleshooting.mdx:109]: thank you!

**Review by laurapacilio — APPROVED** [review]
Approved with one final language nit. Thank you very much!!

**Pull request merged**